src/schema/core.yaml

```diff
@@ -913,6 +913,34 @@ classes:
       - lbc_thirty
       - lbceq
     slot_usage:
+      misc_param:
+        description: Captures key:value pairs that don't align with existing slots or requirements. Inlcude ontology term or standard whenever possible.
+        multivalued: true
+        required: false
+        any_of:
```

Comment:
Thanks. Let's find a way to do this without using any_of. I can help.

Right now, I don't see how those examples would validate, but maybe I'm missing something. 

We should work on identifying which class can or should use the misc_param slots, and build some example data files around them.